docs/source/create/entities/web-entity.rst

```diff
@@ -0,0 +1,130 @@
+##########################
+Play media in a Web Entity
+##########################
+
+Web entities can play a number of different media formats through embedded players.
+
+.. contents:: On This Page
+    :depth: 1
+
+-----------------------
+Supported media formats
+-----------------------
+
+Currently exclusively open source formats are supported:
+
++------------------------+------------------------------------------------------------------+--------------------------+
+| Container format       | Supported codecs                                                 | File extension(s)        |
++========================+==================================================================+==========================+
+| WebM                   | Video: VP8, VP9  Audio: Vorbis, Opus                             | .webm                    |
```

Comment:
Put the "Audio:" section on a new line.
Ditto for similar table content.

Reply:
I don't actually see any way to do this. I just added some more spacing between to make it easier to read and potentially do a new line on its own.